store/postgres/migrations/000020_add_job_dependencies_table.up.sql

```diff
@@ -0,0 +1,9 @@
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+CREATE TABLE IF NOT EXISTS job_dependency (
+    job_id UUID NOT NULL,
+    project_id UUID NOT NULL,
```

Comment:
Should we create a ForeignKey constraint for `project_id`.

Reply:
added for both `project_id` and `dependent_project_id`. also have created index for `job_id` and `project_id`.